### PR TITLE
[TH: Investigations][Serverless & 8.15] New previews in expandable flyout

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/5495

Previews:
- [8.15](https://security-docs_bk_5605.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html) - Revised the preview panel description, updated entity and correlation screenshots, and made minor updates elsewhere.
- [Serverless](https://elastic-dot-co-docs-production-niqi032w8-elastic-dev.vercel.app/current/serverless/security/view-alert-details) - Same changes

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-8pvo3mdpb-elastic-dev.vercel.app

* Built with commit 66ad7273b23da362224c545734e94ec3d9653369

> Issues? Visit #next-docs in Slack

**Review by christineweng — APPROVED** [review]

File: docs/detections/alerts-view-details.asciidoc [hunk 1]
```diff
@@ -52,7 +52,7 @@ IMPORTANT: If you've enabled grouping on the Alerts page, the alert details flyo
 [[preview-panel]]
 === Preview panel 
 
-The preview panel shows more information about the rule associated with the alert and the reason the alert was generated. This panel overlays the right panel when you click **Show rule details** or **Show full reason** in the <<about-section,About>> section. To close the preview panel, click **x**. 
+Some areas in the flyout provide previews when you click on them. For example, clicking **Show rule summary** in the Rule description displays a preview of the rule's details. To close the preview, click **x**.
```
Comment (joepeeples) [inline docs/detections/alerts-view-details.asciidoc:55]: $0.02: I know the UI area is labelled "Rule description," but it doesn't seem so specialized that it needs to be capitalized in general references like this:
```suggestion
Some areas in the flyout provide previews when you click on them. For example, clicking **Show rule summary** in the rule description displays a preview of the rule's details. To close the preview, click **x**. 
```

File: docs/serverless/alerts/view-alert-details.mdx [hunk 2]
```diff
@@ -57,7 +57,7 @@ From the right panel, you can also:
 
 ### Preview panel 
 
-The preview panel shows more information about the rule associated with the alert and the reason the alert was generated. This panel overlays the right panel when you click **Show rule details** or **Show full reason** in the <DocLink slug="/serverless/security/view-alert-details" section="about">About</DocLink> section. To close the preview panel, click **x**. 
+Some areas in the flyout provide previews when you click on them. For example, clicking **Show rule summary** in the Rule description displays a preview of the rule's details. To close the preview, click **x**.
```
Comment (joepeeples) [inline docs/serverless/alerts/view-alert-details.mdx:60]:
```suggestion
Some areas in the flyout provide previews when you click on them. For example, clicking **Show rule summary** in the rule description displays a preview of the rule's details. To close the preview, click **x**. 
```

**Review by joepeeples — DISMISSED** [review]
LGTM! One nit about capitalization but went ahead and approved in case you'd rather keep as-is.

Also, I think "summary" might work better than "preview" in some instances, because "preview" makes me think of the [rule creation preview](https://www.elastic.co/guide/en/security/master/rules-ui-create.html#preview-rules) feature. But the UI uses "preview" so makes sense to stay consistent with that.

**Review by joepeeples — APPROVED** [review]

**Comment by nastasha-solomon** [comment]
Verified that the new flyouts are available in Serverless (huzzah!), so this PR is good to merge!

**Pull request merged**